app/models/slide.rb

```diff
@@ -0,0 +1,6 @@
+class Slide < ApplicationRecord
+  #belongs_to :organization
+  has_one_attached :imageUrl
+  validates :text, presence: true
+  validates :order, presence: true
```

Comment:
add an empty line between validates and active storage

Reply:
Done